apps/lead/filter_set.py

```diff
@@ -5,8 +5,10 @@
 from project.models import Project
 from user.models import User
 from .models import Lead, LeadGroup
-
-
+from organization.models import (
+    Organization,
+    OrganizationType
+)
```

Comment:
Let's import in order like this. (newline in-between)

- Python native modules
- External modules
- Local modules
-Relative modules

So move this before `from .models...`